README.md

```diff
@@ -0,0 +1,9 @@
+## supernova
+[![Build Status](https://travis-ci.org/0xazure/supernova.svg?branch=master)](https://travis-ci.org/0xazure/supernova)
+
+ > Tool for exporting GitHub stars as an organized list.
+
+### things learned
+
+- using mem::[replace](https://github.com/rust-unofficial/patterns/blob/master/idioms/mem-replace.md) to [avoid needless clones](https://github.com/rust-unofficial/patterns/blob/master/idioms/mem-replace.md)
```

Comment:
Something I only learned about `rst` when I tried to make links like this: it doesn't like `::` characters inside the link text.

Since Markdown is more permissive, I'd love to see this finally formatted as [`mem::replace`]().